projects/common/src/telemetry/telemetry.ts

```diff
@@ -0,0 +1,33 @@
+import { InjectionToken, ProviderToken } from '@angular/core';
+import { Dictionary } from './../utilities/types/types';
+export interface UserTraits extends Dictionary<unknown> {
+  email?: string;
+  companyName?: string;
+  name?: string;
+  displayName?: string;
+}
+
+export interface UserTelemetryRegistrationConfig<InitConfig> {
```

Comment:
type params should follow convention (e.g. T, TConfig, TInitConfig, etc.)

Reply:
TInitConfig +1